Buildings/Controls/OBC/Utilities/PIDWithAutotuning/FirstOrderAMIGO.mo

```diff
@@ -148,6 +148,13 @@ protected
       else Buildings.Controls.OBC.CDL.Types.SimpleController.PID
     "Type of controller";
 
+initial equation
```

Comment:
We cannot have the `initial equation` section in OBC package.
Please use the assert block instead.

Reply:
fixed